pipelines/managed/push-artifacts-to-cdn/README.md

```diff
@@ -0,0 +1,21 @@
+# push-artifacts-to-cdn pipeline
+
+Tekton Pipeline to push artifacts to either CDN and/or CGW with signing. 
```

Comment:
Maybe add something like: "It uses InternalRequests so it can be run both in public and private clusters."

I don't think we mentioned that anywhere in the past, but I think it's a useful distinction - for example, the push-binaries-to-dev-portal does not use IRs, so it can only run in private clusters.

Reply:
fixed